### PR TITLE
[WIP] Polyfill Detection

### PR DESCRIPTION
**Pull request by sephcoster**
Short description explaining the high-level reason for the pull request

## Additions

- Updated templates to check browser object support prior to polyfill grab

## Removals

- None

## Changes

- Prior to react initialization, browser detects supported methods for loading polyfills as required

## Testing

- 💯 

## Review

- @JeffreyMFarley 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots

N/A

## Notes

- None

## Todos

- Update method calls to reflect all required methods that might need to be polyfilled.

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: ccdb5_ui/templates/ccdb5_ui/ccdb-main.html [hunk 1]
```diff
@@ -47,15 +47,43 @@
 {% endblock %}
 
 {% block page_js %}
-  <!--[if IE 9]><!-->
-  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=Intl.~locale.en"></script>
-  <!--<![endif]-->
```
Comment (JeffreyMFarley) [inline ccdb5_ui/templates/ccdb5_ui/ccdb-main.html]: You'll want to remove this block from the other file too

File: ccdb5_ui/templates/ccdb5_ui/ccdb-main.html [hunk 2]
```diff
@@ -47,15 +47,43 @@
...
   <script src="{% static "ccdb5.min.js" %}" type="text/javascript"></script>
   <script type="text/javascript">
-    ReactDOM.render(
-      React.createElement(ccdb5_ui.App, {}, null),
-      document.getElementById('root')
-    );
+    function browserSupportsAllFeatures() {
```
Comment (JeffreyMFarley) [inline ccdb5_ui/templates/ccdb5_ui/ccdb-main.html]: FYI: I am moving the bootstrapping code to `index.js` in this PR https://github.com/cfpb/ccdb5-ui/pull/106/files#diff-528c3923d718a8860f5d8c05c3931c55L18

Comment (sephcoster) [reply ccdb5_ui/templates/ccdb5_ui/ccdb-main.html]: OK - I'll wait on that and then update. Thanks!

File: ccdb5_ui/templates/ccdb5_ui/ccdb-main.html (hunk 1 above)
Comment (sephcoster) [reply ccdb5_ui/templates/ccdb5_ui/ccdb-main.html]: 👍

**Pull request closed without merging**